File: Animal.php
<?php

namespace App\Model;

class Animal
{
    public $color = "";
    public $genero = "";
    public $nombre = "";

    //color
    public function __construct($col)
    {
       $this->establecerColor();
    }

    public function obtenerColor()
    {
        return $this->color;
    }

    public function establecerColor($col)
    {
        $this->color = $col;
    }

    //genero
    public function __construct($gen)
    {
       $this->establecerGenero();
    }

    public function obtenerGenero()
    {
        return $this->genero;
    }

    public function establecerEmail($gen)
    {
        $this->genero = $gen;
    }

    //nombre
    public function __construct($nom)
    {
       $this->establecerNombre();
    }

    public function obtenerNombre()
    {
        return $this->nombre;
    }

    public function establecerMarca($nom)
    {
        $this->nombre = $nom;
    }
}

File: Animal.java
package Trabajo;

public class Animal {

    public String nombre;

    public String genero;

    public String color;

    public Animal(String nom, String gen, String col)
    {
        establecerNombre(nom);
        establecerGenero(gen);
        establecerColor(col);
    }

    //nombre
    public String obtenerNombre()
    {
        return nombre;
    }

    public void establecerNombre(String nom)
    {
        nombre = nom;
    }

    //genero
    public String obtenerGenero()
    {
        return genero;
    }

    public void establecerGenero(String gen)
    {
        genero=gen;
    }

    //color
    public String obtenerColor()
    {
        return color;
    }
    public void establecerColor(String col)
    {
        color = col;
    }


    
}
